TypeCobol.Test/Parser/Programs/Cobol85/SubtractionAsUserDefinedWord.rdzMix.txt

```diff
@@ -0,0 +1,152 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZF0OSM.
+       data division.
+       working-storage section.
+        01 a pic 9.
+        01 300-200X  pic 9 value 1.
+        01 200X-100X pic 9 value 1.
+      *KO variable must contains at least one alphabetic char
+Line 9[12,18] <27, Error, Syntax> - Syntax error : Variable name must contain at least one alphabetic char.
+        01 400-500 pic 9.
+
+       procedure division.
+      *Should be KO. 400-500 is not a valid variable name because it
+      *   contains no alphabetic char
+      *   The parser will only report the error on the variable 
+      *   declaration.
+           compute a = 400-500
+
+
+      *KO  "300-200X-100X" was not defined as a data-name.
+      *    The statement was discarded.
+Line 21[24,36] <30, Error, Semantics> - Semantic error: Symbol 300-200X-100X is not referenced
+           compute a = 300-200X-100X.
+      *KO  "-200" was invalid.  Skipped to the next verb, period or
+      *    procedure-name definition.
+      *    A blank was missing before character"X"in column 32.
+      *    A blank was assumed.
+Line 26[28,31] <27, Error, Syntax> - Syntax error : extraneous input '-200' expecting {separator, numeric literal, character string, symbol, statement starting keyword, keyword, Formalized Comments elements, Sql statement starting keyword}
+           compute a = 300 -200X-100X.
+           compute a = 300-200X - 100.
+           compute a = 300 - 200X-100X.
+      *KO  Expected a numeric data item or a numeric literal, but
+      *    found"300-100".  The"COMPUTE"statement was discarded.
+Line 31[12,30] <27, Error, Syntax> - Syntax error : Variable must contains at least one alphabetic char. For subtraction, please add spaces around minus operator: 300 - 100
```

Comment:
The wording is a bit surprising. For a diagnostic message, i would expect something more formal like 'arithmetic operations require spaces around subtraction operator: 300 - 100'

Reply:
Done in e705b29f87cea7244fae092d8bc7488e8de90876.
I choose `minus sign` instead of `subtraction operator` as it's the term used in the Cobol specs.
`Arithmetic operations requires spaces around minus sign: 300 - 100`

Reply:
'require' with no 's'

Reply:
Done in 77275cd8b6d33d153ae576f927f10f78b8e9591b